model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntity.java

```diff
@@ -13,25 +10,28 @@
 import org.neo4j.ogm.typeconversion.UuidStringConverter;
 
 @NodeEntity(label = "cmo_metadb_sample_metadata")
-public class SampleManifestEntity extends SampleManifest {
+public class SampleManifestEntity {
```

Comment:
If this isnt going to be extending SampleManifest anymore then instead of `SampleManifestJsonEntity` we should prob have it as `SampleManifest`.. something that is going to be modeling the data itself and would let us map data fetched from the database easily to a pojo

Reply:
thought about this a bit more, let's:

1. keep the `extends SampleManifest` 
2. remove the fields as well as the getters/setters that you added below for the [`sampleIgoId`](https://github.com/mskcc/cmo-metadb/pull/53/files#diff-f85fc40803e21bbb16de2de8850b9e536810ccc76a0247bf8ca41bb42a06416bR18) and [`sampleInvestigatorId`](https://github.com/mskcc/cmo-metadb/pull/53/files#diff-f85fc40803e21bbb16de2de8850b9e536810ccc76a0247bf8ca41bb42a06416bR20)
3. add back the [`List<Sample> sampleList`](https://github.com/mskcc/cmo-metadb/pull/53/files#diff-f85fc40803e21bbb16de2de8850b9e536810ccc76a0247bf8ca41bb42a06416bL21) as well as its getter/setter
4. add back this [default constructor ](https://github.com/mskcc/cmo-metadb/pull/53/files#diff-f85fc40803e21bbb16de2de8850b9e536810ccc76a0247bf8ca41bb42a06416bL27)

Basically what this will amount to is restoring the file to what it was before but still keep the addition of the `SampleManifestJsonEntity` that you made as well as its getter/setter